jax/api.py

```diff
@@ -1940,15 +1952,17 @@ def make_jaxpr(fun: Callable,
 
   Args:
     fun: The function whose ``jaxpr`` is to be computed. Its positional
-      arguments and return value should be arrays, scalars, or standard Python
       containers (tuple/list/dict) thereof.
+      arguments and return value should be arrays, scalars, or standard Python
     static_argnums: See the :py:func:`jax.jit` docstring.
+      containers (tuple/list/dict) thereof.
```

Comment:
Is there a typo around here, maybe switched lines?

Reply:
yep!